xfinity-data-usage/Dockerfile

```diff
@@ -0,0 +1,2 @@
+ARG BUILD_FROM
+FROM $BUILD_FROM
```

Comment:
A Docker container is build using existing published image. A Dockerfile is required in the add-on directory, so this file cannot be omitted.